packages/bpk-component-horizontal-nav/src/bpk-horizontal-nav-item.scss

```diff
@@ -57,6 +57,7 @@
       @include bpk-themeable-property(color, --bpk-horizontal-nav-link-selected-color, $bpk-horizontal-nav-link-selected-color);
 
       @include bpk-hover {
+        @include bpk-themeable-property(color, --bpk-horizontal-nav-link-selected-color, $bpk-horizontal-nav-link-selected-color);
```

Comment:
So that it doesn't highlight on hover when selected. `disabled` used to take care of this.